_data/compatibility/edge.yaml

```diff
@@ -0,0 +1,83 @@
+---
+name: Edge
+internal_url: /en/compatibility/edge
+logo: /img/compatibility/edge/edge.png
+rbf:
+  tested:
+    date: "2018-10-17"
+    platforms:
+      - iOS
+    version: "1.3.4"
+  features:
+    receive:
+      notification: "false"
+      list: "false"
+      details: "false"
+      shows_replaced_version: "true"
+      shows_original_version: "true"
+    send:
+      signals_bip125: "false"
+      list: "false"
+      details: "false"
+      shows_replaced_version: "na"
+      shows_original_version: "na"
+  examples:
+    - image: /img/compatibility/edge/send-screen-default.png
+      caption: >
+        Sending RBF Transaction - Default Send screen. No option for RBF.
+    - image: /img/compatibility/edge/send-context-fees-menu.png
+      caption: >
+        Sending RBF Transaction - Context menu for send transaction show mining fee options.
+    - image: /img/compatibility/edge/send-change-mining-fee.png
+      caption: >
+        Sending RBF Transaction - Mining fee options dialog.
+    - image: /img/compatibility/edge/send-custom-mining-fee.png
+      caption: >
+        Sending RBF Transaction - Mining fee custom dialog.
+    - image: /img/compatibility/edge/transaction-details-sent.png
+      caption: >
+        Bumping RBF Enabled Transaction - Mining fee custom dialog.
+    - image: /img/compatibility/edge/transaction-details-incoming-rbf.png
+      caption: >
+        Receiving RBF Transaction - Receiving RBF enabled transaction. No RBF Flag.
+    - image: /img/compatibility/edge/transaction-details-incoming-rbf-advanced.png
+      caption: >
+        Receiving RBF Transaction - Transaction details for received RBF transaction. Links to blockchair.com explorer.
```

Comment:
blockchair -> blockchain

Reply:
@adamjonas actually Edge uses the blockchair.com explorer.